Wordle.Api/Wordle.Api/Services/GameService.cs

```diff
@@ -68,8 +68,58 @@ public IQueryable<AllWordStats> StatsForAllWords()
 
         return result;
     }
+
+
+  public async Task<GameStatsDto> SingleWordStats(DateTime date)
```

Comment:
I think this works, but using LINQ would make it look a lot nicer :)

Reply:
I tried to refactor it with linq but ran into a few problems, next time I will use linq on the first implementation.